cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js

```diff
@@ -0,0 +1,49 @@
+var a = 0;
+var b = 1;
+
+Then("User clicks on {string} tab on screen", async (TabName) => {
+    cy.wait(1000);
+    a = 0;
+    b = 1;
+    await cy.xpath("//*[@id='__next']/div[1]/div[2]//a[text()='" + TabName + "']").click();
+})
+
+And('User clicks on {string} button at position {string}', async function (buttonName, position) {
+    // await cy.xpath(Request_SSO_Integration.PlusrequestSSOIntegrationBtn()).click();
+    await cy.xpath("(//button[contains(.,'" + buttonName + "')])[" + position +  "]").click();
+    await cy.wait(2000);
+  });
+
+And("User Enters {string} on {string} textbox at row {string} on {string} pop-up", async (Value, TextboxName, RowNo, PopUpName) => {
+    cy.wait(500);
+    if (TextboxName == 'Role Name') {
```

Comment:
in javascript, strict comparison is preferred; `===` intead of `==`.

Reply:
Done.